xsoar/resource_integration_instance.go

```diff
@@ -270,25 +267,19 @@ func (r resourceIntegrationInstance) Create(ctx context.Context, req tfsdk.Creat
 	var integration map[string]interface{}
 	var httpResponse *http.Response
 	var body []byte
-	err = resource.RetryContext(ctx, 10*time.Minute, func() *resource.RetryError {
-		if plan.Account.Null || len(plan.Account.Value) == 0 {
-			integration, httpResponse, err = r.p.client.DefaultApi.CreateUpdateIntegrationInstance(ctx).CreateIntegrationRequest(moduleInstance).Execute()
-		} else {
-			integration, httpResponse, err = r.p.client.DefaultApi.CreateUpdateIntegrationInstanceAccount(ctx, "acc_"+plan.Account.Value).CreateIntegrationRequest(moduleInstance).Execute()
-		}
+
+	if plan.Account.Null || len(plan.Account.Value) == 0 {
+		integration, httpResponse, err = r.p.client.DefaultApi.CreateUpdateIntegrationInstance(ctx).CreateIntegrationRequest(moduleInstance).Execute()
+	} else {
+		integration, httpResponse, err = r.p.client.DefaultApi.CreateUpdateIntegrationInstanceAccount(ctx, "acc_"+plan.Account.Value).CreateIntegrationRequest(moduleInstance).Execute()
+	}
+	if err != nil {
+		log.Println(err.Error())
```

Comment:
Do we still need to log this if this is already added to the `Diagnostics` of the response?